pandokia/common.py

```diff
@@ -36,33 +36,19 @@ def check_auth():
     # Assume they are not authorized, then look for a reason to allow them in.
     auth_ok = 0
 
-    # If the web server checked them with BasicAuth, we will use that:
```

Comment:
Even though this block is not normally executed in the way that we presently have our installation of Pandokia configured, it is still valid generic code that might be used when spinning up a test installation. 

I suggest putting it back, with a comment explaining that.

Reply:
@vglaidler and I discussed this in person, and decided that this can stay removed.